cdk/src/stack.ts

```diff
@@ -72,9 +72,12 @@ export class HeyAmplifyStack extends Stack {
     // create bucket for SQLite backups with Litestream
     const bucket = new s3.Bucket(this, 'Bucket', {
       bucketName: `${this.appName}-${this.envName}-bucket`,
-      // if env is destroyed, empty & remove bucket
-      removalPolicy: RemovalPolicy.DESTROY,
-      autoDeleteObjects: true,
+      encryption: s3.BucketEncryption.KMS,
```

Comment:
Might be good to set this too: https://docs.aws.amazon.com/cdk/api/v1/docs/aws-s3-readme.html#aws-foundational-security-best-practices

Thinking we also set `bucketKeyEnabled: true` so we get S3 Bucket Keys which save on KMS traffic: https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucket-key.html